pyega3/pyega3.py

```diff
@@ -261,6 +261,10 @@ def download_file_slice(url, token, file_name, start_pos, length, pbar=None):
 
     with requests.get(url, headers=headers, stream=True) as r:
         print_debug_info(url, None, "Response headers: {}".format(r.headers))
+        if r.status_code == 204:
```

Comment:
Even if it's a 204 do we treat this as an exception? :-) Everything looks good to me I just have this question.